cameras:
  - camera_id: 1
    ip: "192.168.0.20"
    port: 554
    username: "admin"
    password: "zjsxmfhf"
    topic_name: "/camera/camera_1/image_raw"
    frame_id: "camera_1_link"
    transport: "tcp"   # or "udp"
    width: 1280
    height: 720

  - camera_id: 2
    ip: "192.168.0.21"
    port: 554
    username: "admin"
    password: "zjsxmfhf"
    topic_name: "/camera/camera_2/image_raw"
    frame_id: "camera_2_link"
    transport: "tcp"
    width: 1280
    height: 720

# Optional: global publisher rate (Hz)
publish_rate: 60.0
